README.md

```diff
@@ -35,7 +39,9 @@ var add = function (x, y) {
 }
 ```
 
--- YOUR ANSWER HERE --
+es5
+no
```

Comment:
This is anonymous. We call this a **function expression**  where a function is assigned to a variable. In this case the function has no name, so it's anonymous.